### PR TITLE
Add note about including remirror styles

### PR DESCRIPTION
**Pull request by StefKors**
### Description

This PR adds a note to the installation docs about including remirror styles.

### Checklist
@ifiokjr Are there any examples that need to be updated?

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [ ] My code follows the code style of this project and `pnpm fix` completed successfully.
- [x] I have updated the documentation where necessary.
- [ ] New code is unit tested and all current tests pass when running `pnpm test`.

### CODE REVIEW

File: docs/guide/installation.md [hunk 1]
```diff
@@ -21,11 +21,22 @@ npm install remirror@next @remirror/pm@next
 You will also notice that you are also installing `@remirror/pm`. This is a peer dependency to all the scoped `@remirror` packages which interact with `prosemirror-*`. It provides some assurances around the versions of prosemirror libraries being used and helps simplify the codebase.
 
 ## Browser support
+Remirror aims to be compatible with all browsers released since **2017**. For this it's required to included the remirror styles.
```
Comment (ifiokjr) [inline docs/guide/installation.md:24]: I think this might fail prettier linting.

File: docs/guide/installation.md [hunk 2]
```diff
@@ -21,11 +21,22 @@ npm install remirror@next @remirror/pm@next
...
+```Javascript
```
Comment (ifiokjr) [inline docs/guide/installation.md:25]: I think it's best to move this section to a new block **Required styles**. It's not really a new or old browser issue. It's more that there are styles which Firefox, and even Chrome require. 

Without the styles on Chrome, whitespace doesn't get added properly. So, you can't insert a <kbd>Space</kbd> character at the end of the document. 

I guess the summary is that it fixes a bunch of browser inconsistencies and is required.

**Review by ifiokjr — APPROVED** [review]

**Comment by ifiokjr** [comment]
> Are there any examples that need to be updated

I think the place you've updated is great. It would be repetitive to include the CSS import in every comment or Markdown code block.

**Comment by StefKors** [comment]
@ifiokjr Added a fixup commit, let me know what you think. if it's all good, feel free to squash or merge the commits into 1

File: docs/guide/installation.md [hunk 3]
```diff
@@ -20,12 +20,31 @@ npm install remirror@next @remirror/pm@next
 
 You will also notice that you are also installing `@remirror/pm`. This is a peer dependency to all the scoped `@remirror` packages which interact with `prosemirror-*`. It provides some assurances around the versions of prosemirror libraries being used and helps simplify the codebase.
 
-## Browser support
+## Required styles
 
-If you are using remirror with a project like [`next.js`](https://nextjs.org/) or [`gatsby`](https://www.gatsbyjs.org/) you will also need to add the following browserlist configuration.
+Remirror requires styles for the editor to function correctly across different browsers. Adding these styles are supported in with the following methods:
+
+```Javascript
```
Comment (ifiokjr) [inline docs/guide/installation.md:27]:
```suggestion
```ts
```

File: docs/guide/installation.md [hunk 4]
```diff
@@ -20,12 +20,31 @@ npm install remirror@next @remirror/pm@next
 
 You will also notice that you are also installing `@remirror/pm`. This is a peer dependency to all the scoped `@remirror` packages which interact with `prosemirror-*`. It provides some assurances around the versions of prosemirror libraries being used and helps simplify the codebase.
 
-## Browser support
+## Required styles
 
-If you are using remirror with a project like [`next.js`](https://nextjs.org/) or [`gatsby`](https://www.gatsbyjs.org/) you will also need to add the following browserlist configuration.
+Remirror requires styles for the editor to function correctly across different browsers. Adding these styles are supported in with the following methods:
```
Comment (ifiokjr) [inline docs/guide/installation.md:25]:
```suggestion
Remirror requires styles for the editor to function correctly across different browsers. These styles can be added in the following ways:
```

File: docs/guide/installation.md [hunk 5]
```diff
@@ -20,12 +20,31 @@ npm install remirror@next @remirror/pm@next
...
+```Javascript
+// Or with emotion
+import { AllStyledComponent } from '@remirror/styles/emotion';
+```
+
+```Javascript
+// Or with styled components
+import { AllStyledComponent } from '@remirror/styles/styled-components';
+```
+
```
Comment (ifiokjr) [inline docs/guide/installation.md:41]:
```suggestion
In order to use the `styled-components` or `emotion` variants you will need to also install `@remirror/styles`.

```

File: docs/guide/installation.md [hunk 6]
```diff
@@ -20,12 +20,31 @@ npm install remirror@next @remirror/pm@next
...
 
-If you are using remirror with a project like [`next.js`](https://nextjs.org/) or [`gatsby`](https://www.gatsbyjs.org/) you will also need to add the following browserlist configuration.
+Remirror requires styles for the editor to function correctly across different browsers. Adding these styles are supported in with the following methods:
+
+```Javascript
+// Plain CSS
+import 'remirror/styles/all.css';
+```
+
+```Javascript
```
Comment (ifiokjr) [inline docs/guide/installation.md:32]:
```suggestion
```ts
```

File: docs/guide/installation.md [hunk 7]
```diff
@@ -20,12 +20,31 @@ npm install remirror@next @remirror/pm@next
...
+// Or with emotion
+import { AllStyledComponent } from '@remirror/styles/emotion';
+```
+
+```Javascript
```
Comment (ifiokjr) [inline docs/guide/installation.md:37]:
```suggestion
```ts
```

**Review by ifiokjr — APPROVED** [review]
I made some automatic suggestions.

**Comment by ifiokjr** [comment]
@StefKors thanks for the update. I've added some automatic suggestions just to tidy things up.

**Pull request merged**